utills/show_user_input.py

```diff
@@ -0,0 +1,61 @@
+from langchain.llms import OpenAI
+from langchain.chains.question_answering import load_qa_chain
+from langchain.callbacks import get_openai_callback
+
+from utills.text_splitter import split_text, create_knowledge_base
+
+
+def retrieve_related_documents(knowledge_base, user_question):
+    """
+        Retrieves related documents from a knowledge base based on a user question.
+
+        Args:
+            knowledge_base: The knowledge base to search for related documents.
+            user_question: The user's question.
+
+        Returns:
+            A list of related documents.
+    """
+    if user_question:
```

Comment:
In structured programming we have one way in and one way out. It makes the code easy to debug if we have just one return statement.